### PR TITLE
Adds Several New Foods

### PR DESCRIPTION
**Pull request by tomixcomics**
-Adding food dishes to accomplish the following goals:
1. Increase the variety of food creation options in game
2. Add more lore based dishes to connect players more to the Lore and try to give more locations their own cuisine identity
3. Utilize some of the less-useful or less utilized ingredients in the game.
-Correcting known issues with existing food already in the game.

Unofficial name: _"The foodening"_.

**New dishes detailed:**

Tagging these foods with the following tags:
(R)=Real dish, that exists in real life
(F)=Fictional dish (as far as I know)
(LB)=Lore Based. Dish is explicitly mentioned and described in Aurora Lore.
(LI)= Lore Inspired. Dish is inspired by existing lore but not explicitly mentioned in it.
(IU)= Ingredient Utilization. Dish uses ingredients less frequently used in the game.

1. Che Troi Nuoc (F) (LB) (IU)
2. North 60 squid (F) (LB) (IU)
3. Sloppy Joe (R)
4. Triolade (R)
5. Qlguabi (F) (LI) (IU)
6. Hengsha rolls (F) (LI)
7. Giffy Pie (F) (IU)
8. Ylpha Roulade (F) (IU)
9. Chocolate Roulade (R)
10. Izu-Ixu (F) (LI) (IU)
11. Pita (R)
12. Falafel Balls (R)
13. Falafel (R)
14. Chocolate Pita (R)
15. Tuna Salad (R)
16. Tuna Pasta Salad (R) (IU)
17. Tuna Salad Pita (R)
18. Sabich (R)
19. Kraken Soup (F) (IU)
20. Box of pralines (R) (which can be obtained in a few secret ways if you're lucky enough or know about them. would love for them to be available for purchase in the space malls/shopping centers as well in the future but not sure how to do that...)
21. Tabbouleh Salad (R) (IU)
22. -Matsuul (F) (IU)
23. -Cake Pops (R) ((Code for cake pops mostly done by CometBlaze, also got some help from dreamy and matt))
24. -K'ois roulade (F)
25. -Floating Island Sorbet (F) (IU)
26. -Cheese Ravioli (R)
27. -Meat Ravioli (R)
28. -Earthenroot Ravioli (F) (IU)
29. -Strawberry Crumble Bars (R) (IU)
30. -Ramen Bowl (R)
31. -Choco-nut spread (R)
32. -Notella bread slice (R)
33. -Sloppy Joe (R)
34. -Spaghetti Bolognese (R)

**Food bug fixes, corrections and changes:**
-Created sprite for condiment bottles of Vanilla Extract
-Reworked pretty much all of the food I helped create from [this update](https://github.com/Aurorastation/Aurora.3/pull/17745) to either fix bugs that it had, differences from the way it was intended to be, create better paths for it, or improve sprites that had issues or were too big compared to other foods.
-Corrected issue with crab meat being invisible in game due to an incorrect path.
-Changed "Bluespace Tomato Soup" to be more in line with what I had in mind when I submitted it back in 2015, to try and make it more rewarding to go through the challenge of making it. Renamed it "Spiral Soup" as was originally intended.
-Corrected issue with the sprite for a meat steak that's only been bitten once not showing up correctly. (Note: Similar bug probably still exists in Xeno-steaks. I have not corrected that one)
-Added new food bounties
-Added new (mostly) food related round start tips.

Additional notes: Various members of the Aurora discord and specifically CometBlaze helped a lot with advice, troubleshooting my code or writing small bits or tweaks to the code when I was struggling with something, for which I am very thankful.

### CODE REVIEW

**Comment by tomixcomics** [comment]
!wip

File: code/modules/reagents/reagent_containers/food/snacks/soup.dm [hunk 1]
```diff
@@ -182,6 +182,17 @@
 	reagent_data = list(/singleton/reagent/nutriment = list("tomato" = 4, "beet" = 4))
 	bitesize = 2
 
+/obj/item/reagent_containers/food/snacks/soup/krakensoup
+	name = "kraken soup"
+	desc = "A zesty Bieselite seafood dish made of squid, pumpkin, paprika and red vegetables. Always a little uncomfortable to eat if there's Skrell around."
```
Comment (listerla) [inline code/modules/reagents/reagent_containers/food/snacks/soup.dm:187]:
```suggestion
	desc = "A zesty Biesellite seafood dish made of squid, pumpkin, paprika and red vegetables. Always a little uncomfortable to eat if there's Skrell around."
```

**Comment by tomixcomics** [comment]
!wip

**Comment by tomixcomics** [comment]
newfoodv3
New foods have all been tested :)
I also have a handy dandy list of recipes and contents ready for the wiki people. (Still need to export all the icons but will send them to you once this is through!)

**Comment by tomixcomics** [comment]
!review

**Review by Alberyk — APPROVED** [review]

**Comment by tomixcomics** [comment]
Agghhh, okay, before this gets further approval or merging - I found a big bug in this update that I don't know how to solve, any help in solvinig it would be appreciated. Basically - now when you use mixing bowls but mix things that don't actually create any recipe, it gives a runtime error and creates a phantom object called "obj". I am clueless about how to fix this - can someone please help?

File: code/modules/cooking/recipes/recipes_fryer.dm [hunk 2]
```diff
@@ -167,3 +167,18 @@
...
 	result = /obj/item/reagent_containers/food/snacks/corn_dog
+
+/singleton/recipe/north60squid
+	appliance = FRYER
+	items = list(/obj/item/reagent_containers/food/snacks/squidmeat = 1)
+	reagents = list(/singleton/reagent/drink/lemonjuice = 5 , /singleton/reagent/drink/applejuice = 5, /singleton/reagent/nutriment/garlicsauce = 10)
+	coating = /singleton/reagent/nutriment/coating/beerbatter
+	reagent_mix = RECIPE_REAGENT_REPLACE //Simplify end product
+	result = /obj/item/reagent_containers/food/snacks/north60squid
+	//XANAN
```
Comment (FluffyGhoster) [inline code/modules/cooking/recipes/recipes_fryer.dm:178]: ?

File: code/modules/reagents/reagent_containers/food/sandwich.dm [hunk 3]
```diff
@@ -89,3 +89,6 @@
 	bitesize = 2
 	base_name = "burger"
 	topper = "burger_top"
+
+
+
```
Comment (FluffyGhoster) [inline code/modules/reagents/reagent_containers/food/sandwich.dm:94]:
```suggestion
```

File: code/modules/reagents/reagent_containers/food/snacks/cultural/vaurca.dm [hunk 4]
```diff
@@ -174,13 +174,39 @@
...
+
+/obj/item/reagent_containers/food/snacks/koisrouladeslice
+	name = "k'ois roulade slice"
+	desc = "K'ois, with a twist!"
+	icon = 'icons/obj/item/reagent_containers/food/cultural/vaurca.dmi'
+	icon_state = "koisrouladeslice"
+	trash = /obj/item/trash/plate
+	filling_color = "#E6E600"
+	bitesize = 5
+/obj/item/reagent_containers/food/snacks/vkrexiwrap/meat
```
Comment (FluffyGhoster) [inline code/modules/reagents/reagent_containers/food/snacks/cultural/vaurca.dm:198]:
```suggestion

/obj/item/reagent_containers/food/snacks/vkrexiwrap/meat
```

File: config/example copy/access levels.txt [hunk 5]
```diff
@@ -0,0 +1,79 @@
+HOW TO CONVERT A MAP TO THE NEW (june 2008) ACCESS LEVEL SYSTEM
```
Comment (FluffyGhoster) [inline config/example copy/access levels.txt:1]: This folder should not be part of this PR

**Review by FluffyGhoster — DISMISSED** [review]

File: code/modules/cooking/recipes/recipes_fryer.dm (hunk 2 above)
Comment (tomixcomics) [reply code/modules/cooking/recipes/recipes_fryer.dm:178]: What's confusing? The annotation at the end? That's my bad, I thought it was a way of designating the origin of a dish before I realized how it's actually categorized.

File: config/example copy/access levels.txt (hunk 5 above)
Comment (tomixcomics) [reply config/example copy/access levels.txt:1]: I... Don't even know what this is or how it ended up in my PR? I didn't touch anything that has anything to do with access. How do I remove it form the PR? click "delete file"? or will that make the github think i want this file deleted as part of the merge?

Comment (FluffyGhoster) [reply config/example copy/access levels.txt:1]: Delete the example copy folder on your PC inside this branch, commit

Comment (tomixcomics) [reply config/example copy/access levels.txt:1]: Okay, did that and committed. though now the "push" button changed to "pull origin". Do I need to do anything? Force push? (Sorry, I'm still new at this.)

File: code/modules/cooking/recipes/recipes_fryer.dm (hunk 2 above)
Comment (tomixcomics) [reply code/modules/cooking/recipes/recipes_fryer.dm:178]: Fixed. Thank you!

File: config/example copy/access levels.txt (hunk 5 above)
Comment (FluffyGhoster) [reply config/example copy/access levels.txt:1]: Try to pull origin, then it should let you push

Comment (tomixcomics) [reply config/example copy/access levels.txt]: Okay, done! Thank you! Hope that fixes it!

**Review by NonQueueingMatt — APPROVED** [review]
